src/tests/ftest/scripts/main.sh

```diff
@@ -256,9 +256,18 @@ fi
 # shellcheck disable=SC2086
 export WITH_VALGRIND
 export STAGE_NAME
+
+launch_node_args = "-ts \"${TEST_NODES}\""
```

Comment:
(lint) Remove spaces around = to assign (or use [ ] to compare, or quote '=' if literal). [SC2283]